<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IsaWidget</class>
 <widget class="QWidget" name="IsaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="controls_layout_" stretch="0,1,0,0,0,0">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignmentFlag::AlignTop">
      <widget class="QWidget" name="viewing_options_widget_" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="viewing_options_widget_layout_">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignVCenter">
         <widget class="ArrowIconComboBox" name="viewing_options_combo_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Viewing Options</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Resources/assets/PullDownOff_Gray.svg</normaloff>:/Resources/assets/PullDownOff_Gray.svg</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="viewing_options_checkboxes_widget_" native="true">
          <layout class="QGridLayout" name="viewing_options_checkboxes_layout_">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLineEdit" name="go_to_line_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Go to line...</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="TextSearchWidget" name="search_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="search_results_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>No results</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="IsaBranchLabelNavigationWidget" name="branch_label_navigation_" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="IsaTreeView" name="isa_tree_view_">
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ArrowIconComboBox</class>
   <extends>QPushButton</extends>
   <header>qt_common/custom_widgets/arrow_icon_combo_box.h</header>
  </customwidget>
  <customwidget>
   <class>TextSearchWidget</class>
   <extends>QLineEdit</extends>
   <header>qt_common/custom_widgets/text_search_widget.h</header>
  </customwidget>
  <customwidget>
   <class>IsaBranchLabelNavigationWidget</class>
   <extends>QWidget</extends>
   <header>qt_isa_gui/widgets/isa_branch_label_navigation_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IsaTreeView</class>
   <extends>QTreeView</extends>
   <header>qt_isa_gui/widgets/isa_tree_view.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../frontend/resources.qrc"/>
 </resources>
 <connections/>
</ui>
